src/sidebar/config/service-config.js

```diff
@@ -11,9 +11,9 @@
  * @return {Service|null}
```

Comment:
My guess is we can do away with the return value here

Reply:
You are right, it could be inferred.